src/fs.h

```diff
@@ -12,7 +12,6 @@
 #endif
 
 #include <boost/filesystem.hpp>
-#include <boost/filesystem/fstream.hpp>
```

Comment:
Next time you're removing a header please check carefully that it's not used; we're also using includes to signal dependencies; so it being included indirectly, somehow, is not a good reason to remove it at the direct use site.